replicate/common/consts.go

```diff
@@ -12,4 +12,5 @@ const (
 	ReplicateToMatching             = "replicator.v1.mittwald.de/replicate-to-matching"
 	KeepOwnerReferences             = "replicator.v1.mittwald.de/keep-owner-references"
 	StripLabels                     = "replicator.v1.mittwald.de/strip-labels"
```

Comment:
Maybe add a `ForceOverwrite: true` to explicitly overwrite existing Services, so we do not overwrite existing Services by accident?

Reply:
will handle this in a separate PR as it changes the logic of the current software